pytket/binders/circuit/Circuit/add_classical_op.cpp

```diff
@@ -237,20 +237,24 @@ void init_circuit_add_classical_op(
           "add_c_setreg",
           [](Circuit &circ, const _tket_uint_t value, const BitRegister &reg,
              const py::kwargs &kwargs) {
+	    if (value >> reg.size() != 0) {
+	      throw std::runtime_error("Value " STR(value) " cannot be held on a " STR(reg.size()) "-bit register."); 
```

Comment:
This `STR` doesn't work here; I think if you want to include the values in the string you have to explicitly construct the string from a stringstream.

Reply:
Ditto, in the last commit.